packages/Confirmation/src/Confirmation.js

```diff
@@ -121,6 +121,7 @@ const Confirmation = props => {
               kind={confirmButtonType}
               size={buttonSize}
               onClick={handleOnConfirm}
+              data-qa-anchor="confirmation__confirm-button"
```

Comment:
Wouldn't this be `data-pka-anchor` (with the appropriate naming convention)? I think `data-qa-anchor` is something apps use, not something that should be in paprika.